deployments/values/supported-matrix.yaml

```diff
@@ -204,6 +217,9 @@ matrix:
     acs:
       version: "7.2"
       pattern: *ga_hotfixes_pattern
+    activemq:
+      version: "~5.16.1"
+      pattern: -jre17-rockylinux8
```

Comment:
```suggestion
      version: "5.16"
      pattern: *ga_activemq_pattern
```

was a test?

Reply:
I looked at doc https://docs.alfresco.com/content-services/7.2/support/ and left it like this. Will fix it.